ssc/src/Pos/Security/Params.hs

```diff
@@ -61,8 +61,10 @@ instance A.FromJSON AttackType where
             ", acceptable values are NoBlocks|NoCommitments"
 
 data AttackTarget
-    = NetworkAddressTarget { attNetworkAddr :: NetworkAddress}
-    | PubKeyAddressTarget { attPkAddr :: StakeholderId}
+    -- NetworkAddressTarget attNetworkAddr
```

Comment:
I actually feel like these constructor names are self-explanatory enough, but if we'd still like to document them (which I'm not opposed to), perhaps we should use Haddock annotations and expand a bit more on the descriptions.

Reply:
Agreed. These are pretty straightforward, so dropping the comments seems fine to me. If there's more explanation needed then Haddock comment would be good.